doc/whats-new.rst

```diff
@@ -3,6 +3,15 @@
 What's New
 ===========
 
+v0.6.0 (unreleased)
+-------------------------
+
+Bug fixes
+~~~~~~~~~
+- Fix numba related bug replacing :py:meth:`np.flip` with indexing :py:meth:`[::-1]` (:pull:`263`, :issue:`262`). By `Navid Constantinou <https://github.com/navidcy>`_.
```

Comment:
Just one nitpick: I would prefer a more general description like: "Adding support for older numba versions (<0.49)"

Reply:
Your suggestion is actually much better.